src/Backend/Modules/Partners/Domain/Widget/WidgetType.php

```diff
@@ -52,4 +70,30 @@ public function configureOptions(OptionsResolver $resolver)
             ]
         );
     }
+
+    /**
+     * Get templates.
+     *
+     * @return array
+     */
+    private function getPossibleTemplates()
+    {
+        $templates = array();
+        $finder = new Finder();
+        $finder->name('*.html.twig');
+        $finder->in(FRONTEND_MODULES_PATH . '/Partners/Layout/Widgets');
+        // if there is a custom theme we should include the templates there also
+        $theme = Model::get('fork.settings')->get('Core', 'theme', 'core');
```

Comment:
in principle you should inject fork.settings via the constructor and create a service of the form but I'm willing to let this one slip